src/main/java/at/hannibal2/skyhanni/features/rift/everywhere/Ubikreminder.kt

```diff
@@ -0,0 +1,37 @@
+package at.hannibal2.skyhanni.features.rift.everywhere
+
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.features.rift.RiftAPI
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ChatUtils
+import at.hannibal2.skyhanni.utils.DelayedRun
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import kotlin.time.Duration.Companion.hours
+
+@SkyHanniModule
+object Ubikreminder {
+
+    private val config get() = RiftAPI.config.ubikReminder
+
+    private var isTimerRunning = false
+    private val messageRegex = Regex("ROUND [1-9] \\(FINAL\\):")
```

Comment:
please use our repo patterns system instead. for an exampe, see https://github.com/hannibal002/SkyHanni/blob/d4510fa75a26cdadb9cdeedfd29c89201f0b8fce/src/main/java/at/hannibal2/skyhanni/features/slayer/SlayerProfitTracker.kt#L53